_im-plugin/ism/policies.md

```diff
@@ -558,9 +558,11 @@ The following sample template policy is for a rollover use case.
    PUT _index_template/ism_rollover
    {
      "index_patterns": ["log*"],
-     "settings": {
-       "plugins.index_state_management.rollover_alias": "log"
-     }
+     "template": {
+      "settings": {
+       "opendistro.index_state_management.rollover_alias": "log"
+      }
```

Comment:
Do we need to replace "opendistro" with "plugins"?

Reply:
verified both of the requests work.
GET _opendistro/_ism/explain/log-000001?pretty

GET _plugins/_ism/explain/log-000001?pretty

But to be consistent I changed opendistro to plugins